src/Compilers/VisualBasic/Test/Syntax/Parser/ParseErrorTests.vb

```diff
@@ -1035,7 +1035,6 @@ End Module
 
         ParseAndVerify(code, <errors>
                                  <error id="30081"/>
-                                 <error id="30198"/>
                              </errors>)
```

Comment:
Oh, I see, it is a valid syntax form now.

---
In reply to: [70129632](https://github.com/dotnet/roslyn/pull/12391#discussion_r70129632) [](ancestors = 70129632)